src/__tests__/lib/containers/GenericCard.test.tsx

```diff
@@ -26,6 +26,7 @@ import {
   ShowMore,
 } from '../../../lib/containers/row_renderers/utils'
 import IconSvg from '../../../lib/containers/IconSvg'
+import { server } from '../../../mocks/msw/server'
```

Comment:
Adding MSW also fixed a lot of "memory leaks". It might be that some components are rendering that hit the real Synapse backend, and the latency/type of failure causes an asynchronous issue.